app/index.html

```diff
@@ -27,9 +27,14 @@
   <script src="/js/lib/chosen-angular.js"></script>
   <script src="//cdn.tinymce.com/4/tinymce.min.js"></script>
   <script src="/js/lib/angular-ui-tinymce.js"></script>
-
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.11/moment-timezone.js"></script>
```

Comment:
is there a reason you're using a cdn for `moment-timezone`, but storing `moment` in `/js/lib`?

Reply:
also, would `https://github.com/urish/angular-moment` be more suitable?